table/queries/merged_reduced_scans.sql

```diff
@@ -67,14 +71,16 @@ WITH AllScans AS (
     WHERE NOT controls_failed
     GROUP BY date, source, country_code, network, outcome, domain, category, subnetwork
     # Filter it here so that we don't need to load the outcome to apply the report filtering on every filter.
-    HAVING NOT STARTS_WITH(outcome, "❔setup/")
+    HAVING (NOT STARTS_WITH(outcome, "❔setup/")
+            AND NOT outcome = "❔read/system")
 )
 SELECT
     Grouped.* EXCEPT (country_code),
     IFNULL(country_name, country_code) AS country_name,
     CASE
         WHEN (STARTS_WITH(outcome, "✅")) THEN 0
         WHEN (STARTS_WITH(outcome, "❗️")) THEN count
+        WHEN (STARTS_WITH(outcome, "❓")) THEN count
```

Comment:
What is the reasoning behind counting these cases? Ideally, we only want to count cases that we're more sure about.

Reply:
We still consider those results as unexpected, so we count them as unexpected.
It's just that the expectation is not super clear with those.

Reply:
But maybe they lead to certain dubious cases bubbling up, when it may actually be worthwhile to bubble up ones we're more confident about. What about creating a filter where the count can be generated dynamically based on the filter?

Reply:
Counting these differently would be a very large change to the way we're displaying the data in the pivot table. I'm not fully opposed to it. But I wouldn't want to do it as part of this change. It would have to be its own thing with its own separate analysis.